src/pages/overview/document-generation-api/templatetags.md

```diff
@@ -284,6 +275,61 @@ over the specified array of objects.
 
 ## Lists
 
+### Insert List using `<ul>` and `<ol>` HTML Elements
+Add lists in the document through JSON data using `<ul>` (unordered list) and `<ol>` (ordered list) tags.
+
+**Simple Lists**
+
+JSON representation of the input data:
+
+```json
+{
+  "List_Data": "<ul><li>Coffee</li><li>Tea</li><li>Milk</li></ul>"
+}
+```
+
+Placeholder tag is replaced by a list generated using the html string provided in the input json.
+
+![Sample for HTML Table insertion](../images/List_Example_1.png)
+
+**Nested Lists**
+
+JSON representation of the input data:
+
+```json
+{
+  "List_Data": "<ol><li>Level 1</li><li>Level 1<ol><li>Level 2</li><li>Level 2<ol><li>Level 3</li><li>Level 3</li></ol></li></ol></li></ol>"
+}
+```
+
+Placeholder tag is replaced by a list generated using the html string provided in the input json.
+
+![Sample for HTML Table insertion](../images/Nested_Numbered_Lists_Example.png)
+
+<InlineAlert slots="text"/>
+
+- Placeholder Tags containing `<ul>` or `<ol>` HTML elements as values in input JSON data should be placed in a separate paragraph in the input document template.
+
+- Only Inline Styling of List elements is possible. i.e. using style attribute of `<ol>` & `<ul>`. Internal and External styling are not allowed.
```

Comment:
possible -> allowed.

Remove Internal and External Styling line as suggested by Ray.

Reply:
Done